extensions/interactions/CodeRepl/directives/CodeReplValidationServiceSpec.js

```diff
@@ -27,7 +27,7 @@ describe('CodeReplValidationService', function() {
     currentState = 'First State';
     goodDefaultOutcome = {
       dest: 'Second State',
-      feedback: []
+      feedback: null
```

Comment:
As before -- is null correct? Ditto in the other *ServiceSpec.js files.

Reply:
fixed